controllers/student/profile/profile.controller.js

```diff
@@ -0,0 +1,16 @@
+const model = require('../../../models/index');
+
+// Return information information about the logged in user
+exports.profileData = (req, res) => {
+  model.User.findOne({
+    where: { external_id: req.session.external_id },
+    include: [
+      { model: model.GroupAllocation },
+      { model: model.PreferenceSelection }
+    ]
+   }).then(user => {
+     res.send(user);
+   }).catch(err => {
+     res.status(500).send({ message: err.message || 'Something went wrong' })
+   })
+}
```

Comment:
Works, but also works with admins, not just students. Need to be changed or remain?

Reply:
that's fine as it is, we can always update if we wish to change the logic